blkn/src/Button/IconWrapper.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+import * as React from "react";
+import { colors } from "../constants";
+
+const sizes = {
+  normal: "20px",
+  small: "20px",
+  large: "26px",
+};
+
+const fillColors = {
+  primary: colors.white,
+  secondary: colors.shuttle,
+};
+
+type Props = {
+  size: $Keys<typeof sizes>,
+  type: $Keys<typeof fillColors>,
+  icon: React.StatelessFunctionalComponent<*>,
```

Comment:
`Icon: React.ComponentType<any>`

Reply:
Tried that already before. Doesn't work...

Reply:
plz elaborate. `React.StatelessFunctionalComponent<any>` is a subtype of `React.ComponentType<any>`, it _has_ to work